project-koku/koku
# simplify category filtering

## Jira Ticket

[COST-3268](https://issues.redhat.com/browse/COST-3268]

## Description

This change will enable you to filter on `platform` category and magically filter the projects that make up this category.

## Testing

1. Checkout Branch
2. Restart Koku
3. Load data with and w/o openshift-{something} and kube-{something} project names
4. group_by[project]=*&filter[project]=platform should yield all projects matching openshift- and kube-
5. Vise Versa for exclude here.

## Notes

...
